cirq/ops/gate_features.py

```diff
@@ -49,6 +46,8 @@ def on_each(self, *targets: Union[raw_types.Qid, Iterable[Any]]) -> List[raw_typ
         Raises:
             ValueError if targets are not instances of Qid or List[Qid].
```

Comment:
We should change the raises docstring to cater for the "times" when it's not a single qubit gate

Reply:
Done.